conventions/Documentation.md

```diff
@@ -1,20 +1,67 @@
+- Only use single quotes (`'`) as apostrophes.
+- No links or URLs allowed in descriptions. If a link is necessary, a [custom JSDoc tag](https://stenciljs.com/docs/docs-json#custom-jsdocs-tags) should be added and parsed in the SDK site.
+- Refrain from using "e.g." or "i.e." references. Leverage "such as" (or similar) where examples are referenced.
+- Use "Accessible" instead of "Aria" or "a11y" language.
+- Verify slots and properties/attributes don't use the text "optional" in their descriptions.
+
+### Deprecation notices
+
+There are two ways to document deprecations, depending on the API reference. In both cases a deprecated chip will be displayed in the SDK site within the component's API reference section.
+
+1. The `@deprecated` JSDoc tag is used for JavaScript properties, events, and methods in the `<component-name>.tsx` file. Notes can accompany the JSDoc tag, such as "use `<property>` instead".
+2. The `[Deprecated]` text is added at the beginning of the JSDoc description for slots (`@slots`) in the `<component-name>.tsx` file and CSS variables in the `<component-name>.scss` file. The text is parsed and removed from the description in the SDK site.
+
+### Usage snippets
+
+You can provide code snippets demonstrating a specific behavior or pattern for a component. Within the component's directory, create a new `usage` directory. Then, create a Markdown file with the filename as the title of the snippet. There should only be one snippet per Markdown file. Stencil will add the usage snippets to the component's README after building. These usage snippets will then be displayed in Storybook.
+
+## Storybook
+
 Calcite Components uses [Storybook](https://storybook.js.org/) to provide an interactive showcase of components with accompanying documentation.
 
-For each main component (i.e., one that can be used by itself), there should be a `<component-name>.stories.ts` file in its component folder.
+For each main component (i.e. one that can be used by itself), there should be a `<component-name>.stories.ts` file in its component folder.
 
 Each story should provide access to relevant [knobs](https://github.com/storybookjs/storybook/tree/next/addons/knobs) so users can test out different properties.
 
-For additional documentation, create a [usage folder](https://github.com/Esri/calcite-components/tree/master/src/components/action/usage) in the component directory with a basic.md and optionally an advanced.md file (if additional documentation or examples are required) with snippets showing different supported use cases for the component.
+### Writing stories
+
+Each component should use a `storiesOf` with at least one story. If your component has properties that effect visual styles, you can use the [storybook knobs addon](https://www.npmjs.com/package/@storybook/addon-knobs) to allow people to manipulate properties and see live updates in the documentation. A minimal stories file might look something like this:
```

Comment:
The `storiesOf` API [is considered legacy](https://github.com/storybookjs/storybook/blob/master/lib/core/docs/storiesOf.md#storiesof-legacy-api). Stories should follow [CSF](https://storybook.js.org/docs/html/api/csf).